source/Symbol/SwiftASTContext.cpp

```diff
@@ -3862,9 +3863,10 @@ SwiftASTContext::GetTypeFromMangledTypename(const char *mangled_typename,
 
   if (!mangled_typename ||
       !SwiftLanguageRuntime::IsSwiftMangledName(mangled_typename)) {
-    error.SetErrorStringWithFormat("typename '%s' is not a valid Swift mangled "
-                                   "typename, it should begin with $S",
-                                   mangled_typename);
+    error.SetErrorStringWithFormat(
+        "typename '%s' is not a valid Swift mangled "
+        "typename, it should begin with " MANGLING_PREFIX_STR,
+        mangled_typename);
```

Comment:
This error message is way too specific. It should just say ... is not a valid Swift mangled name. No consumer will read this error and think "Oh, I need to change the prefix!" :-)